### DISCUSSION TITLE
How to install Carbon on any OS ?

### DISCUSSION THREAD

**Discussion by adhavan-m** [carbon-language/carbon-lang#4617] [Getting started]
I am about to start experimenting Carbon language and want to write simple code, compile and execute them. I can find the current version of toolchain here: https://github.com/carbon-language/carbon-lang/releases/tag/v0.0.0-0.nightly.2024.12.03 but unable to find a binary distribution of the language itself which installs like any other language like Go, Rust, etc. Please guide me on how to install Carbon on any Operating System.

**Comment by jonmeow** [comment]
Carbon is still in early development. The project status is [detailed here](https://github.com/carbon-language/carbon-lang/?tab=readme-ov-file#project-status). If you want to try out the current state, see [getting started](https://github.com/carbon-language/carbon-lang/?tab=readme-ov-file#getting-started). We aren't prioritizing cross-OS install setups until Carbon is more complete. It may be a little too early for your interest.
